tests/cases/fourslash/quickInfoInheritedJSDoc.ts

```diff
@@ -1,14 +0,0 @@
-/// <reference path='fourslash.ts'/>
-// #32708
-
-////interface I<T> {
-////    /** only once please */
-////    t: T
-////}
-////interface C<T> extends I<T> {
-////    t: T
-////}
-////declare var cnsb: C<number> & C<string> & C<boolean>;
-////cnsb.t/**/
-
-verify.quickInfoAt("", "(property) C<T>.t: never", "only once please");
```

Comment:
Since I think this test is here to validate that quick info isn't duplicated, validating that there's no quickinfo at this location anymore might be better than deleting the test (at least from the perspective of keeping an eye toward regressions).

Reply:
What's the fourslash incantation needed to verify there's no quick info?